assets/erc-7291/contracts/IPBM_AddressList.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Same license concern.

Reply:
Licenses of sample code have been changed to CC0-1.0